DSCResources/MSFT_xDNSServerAddress/MSFT_xDNSServerAddress.psm1

```diff
@@ -55,11 +59,52 @@ function Set-TargetResource
         [String]$InterfaceAlias,
 
         [Parameter(Mandatory)]
-        [ValidateSet("IPv4", "IPv6")]
+        [ValidateSet('IPv4', 'IPv6')]
         [String]$AddressFamily
     )
 
-    Test-Properties @PSBoundParameters -Apply
+    try
+    {        
+        Write-Verbose -Message ( @("$($MyInvocation.MyCommand): "
+            'Applying the DNS Server Address ...'
+            ) -join '')
+
+        #Should not need to validate the settings because Test-TargetResource already did this
```

Comment:
While this comment makes sense in this review, I'm not sure it actually adds any value in the long term. Could we remove it?